parsec/_parsec_pyi/regex.pyi

```diff
@@ -4,7 +4,9 @@ from __future__ import annotations
 
 class Regex:
     @classmethod
-    def from_pattern(cls, pattern: str) -> Regex: ...
+    def from_file(cls, pattern_file_content: str) -> Regex: ...
```

Comment:
```suggestion
    def from_file(cls, path: Path) -> Regex: ...
```

Reply:
`std::path` cannot be used by `pyo3` because it does not implements `PyClass` trait.

Reply:
See https://github.com/Scille/parsec-cloud/pull/3737#discussion_r1044221291

Reply:
just use `path: str` the biggest problem here is the name of the param